sql_user_check_redshift/README.md

```diff
@@ -0,0 +1,19 @@
+# Redshift User Check
+
+database_stats.sh script can be used to test connectivity with Redshift database as well as fetch some stats and metadata. 
+The script supports 2 modes which can be specified with -m flag while running the script. One mode is stats mode which 
+will just print info like list of databases with their size. Count of tables, rows, column, size per schema. 
+Distribution of data by data type. Stats mode is the default mode. Another mode is full_metadata which will print 
+complete information of all columns like their data type, constraints etc.
+
+From `sql_user_check_redshift` directory, run these commands depending on mode of script. 
```

Comment:
Mention here that psql must be installed. Also add instructions or a link to how to install it

Reply:
done